about.html

```diff
@@ -408,7 +409,7 @@ <h4 class="title">
 =======
                             Over 25 years working in tax services developing x  and x for
 >>>>>>> origin/about-page-michael
```

Comment:
https://github.com/codesydney/aicta/blob/804b53d1e289ae5f5cf4f182459f5d5b29fe1e19/about.html#L407-L411

Opps, looks like there was a merge conflict marker. Sorry, I should have picked this up last time.

Can you please do me a favour and fix up the lines above?

Reply:
hi dave,should be resolved the conflict.just check it.